core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1808,6 +1810,33 @@ public ServerBuilder http1HeaderNaming(Http1HeaderNaming http1HeaderNaming) {
         return this;
     }
 
+    /**
+     * Sets the interval between log messages that report the
+     * exceptions not logged by any decorators or services, such as
+     * {@link LoggingService}.
+     * @param unloggedExceptionReportInterval the interval between the summary log messages, or
+     *                                        {@link Duration#ZERO} to disable this feature
+     * @throws IllegalArgumentException if specified {@code unloggedExceptionReportInterval} is negative.
+     */
+    public ServerBuilder unloggedExceptionReportInterval(Duration unloggedExceptionReportInterval) {
```

Comment:
I think using "unhandled" might be easy for users understanding because users can handle exceptions not by logging but also in many different ways(even though we are only considering `LoggingService` in this case). 
But I also think that we are only considering `LoggingService` on this issue, using `unlogged` might be more explicit on conveying information. 

How do you think?? @trustin

Reply:
Yeah, I thought it sounds a little bit awkward as well. Maybe `unhandled` is fine.